iib/workers/tasks/build.py

```diff
@@ -812,15 +817,25 @@ def handle_add_request(
             log.info('Building a temporary index image to satisfy the deprecation requirement')
             _build_image(temp_dir, 'index.Dockerfile', request_id, arch)
             intermediate_image_name = _get_local_pull_spec(request_id, arch, include_transport=True)
-            deprecate_bundles(
-                deprecation_bundles,
-                temp_dir,
-                prebuild_info['binary_image'],
-                intermediate_image_name,
-                overwrite_from_index_token,
-                # Use podman so opm can find the image locally
-                container_tool='podman',
-            )
+            if is_fbc:
```

Comment:
Can we remove lines 810-813 in favor of this one?

Reply:
:facepalm: Thanks!